docs/tutorial/airbyte_config_persistence.md

```diff
@@ -0,0 +1,518 @@
+
+```
+-v, --volumes           Remove named volumes declared in the `volumes`
+                            section of the Compose file and anonymous volumes
+                            attached to containers.
+```
+
+Note that the `/tmp/airbyte_local:/local` that we saw earlier is a [bind mount](https://docs.docker.com/storage/bind-mounts/) so data that was replicated locally won't be affected by the next command.
+
+However it will get rid of the named volume workspace so all logs and generated files by Airbyte will be lost.
+
+We can then run:
+
+
+
+```bash
+docker-compose down -v
+docker-compose up -d
+```
+
+    WARNING: The API_URL variable is not set. Defaulting to a blank string.
+    Stopping airbyte-webapp    ... 
+    Stopping airbyte-scheduler ... 
+    Stopping airbyte-server    ... 
+    Stopping airbyte-db        ... 
+    Removing airbyte-webapp    ... mdone
+    Removing airbyte-scheduler ... 
+    Removing airbyte-server    ... 
+    Removing airbyte-data-seed ... 
+    Removing airbyte-db        ... 
+    Removing init              ... 
+    Removing network airbyte_defaultdone
+    Removing volume airbyte_workspace
+    Removing volume airbyte_data
+    Removing volume airbyte_db
+    WARNING: The API_URL variable is not set. Defaulting to a blank string.
+    Creating network "airbyte_default" with the default driver
+    Creating volume "airbyte_workspace" with default driver
+    Creating volume "airbyte_data" with default driver
+    Creating volume "airbyte_db" with default driver
+    Creating init ... 
+    Creating airbyte-data-seed ... 
+    Creating airbyte-db        ... 
+    Creating airbyte-server    ... mdone
+    Creating airbyte-scheduler ... mdone
+    Creating airbyte-webapp    ... mdone
+    ting airbyte-webapp    ... done
+
+Wait a moment for the webserver to start and go refresh the page http://localhost:8000/.
+
+We are prompted with the onboarding process again...
+
+Let's ignore that step, close the page and go back to the notebook to import configurations from our initial setup instead.
+
+## Restore our initial setup
+
+We can play and restore files in the named docker volume `data` and thus retrieve files that were created from earlier:
+
+
+
+```bash
+docker cp $TUTORIAL_DIR/my-setup/data/config airbyte-server:data
+```
+
+Now refresh back the page http://localhost:8000/ again, wait a little bit for the server to pick up the freshly imported configurations...
+Tada! We don't need to complete the onboarding process anymore! 
+and we have the list of connectors that were created previously available again. Thus you can use this ability of export/import files from named volumes to share with others the configuration of your connectors.
+
+Warning: and it will include credentials, so be careful too!
+
+## Configure some Covid (data) source and Postgres destinations
+
+Let's re-iterate the source and destination creation, this time, with a file accessible from a public API:
```

Comment:
Is there a global file for that instead of France?

Reply:
There is one (and that's what we use in our integration tests) but this one is actually from a question from a user in slack that couldn't make it work with this particular file from this public API from french data.gouv.fr ...